test/vars/MavenSpec.groovy

```diff
@@ -101,4 +101,14 @@ class MavenSpec extends JenkinsPipelineSpecification {
         then:
         1 * getPipelineMock("sh")('mvn -B -s settingsFileId -fae clean install')
     }
+
+    def "[maven.groovy] update pom element"() {
+        setup:
+            def pomURL = "https://raw.githubusercontent.com/kiegroup/kogito-runtimes/master/pom.xml"
+            def pom = new URL(pomURL).getText();
+        when:    
+            mavenGroovy.updatePomElement(pom,"version.maven","0.0.0")
+        then:
+            notThrown(MissingPropertyException)
```

Comment:
A pending assert should be added
```
1 * getPipelineMock('writeFile')(pom, _) // _ (which means whatever) since the result of updatePomElement will be tested by another test
```

Reply:
Hey @Ginxo   For some reason it doesn't work, maybe I'm doing something wrong.